### DISCUSSION TITLE
Dashboard Side menu

### DISCUSSION THREAD

**Discussion by prasadjayasinghe84** [nasirkhan/laravel-starter#603] [Q&A] [answered]
Earlier, when we created a module, the module name was added to the admin dashboard side menu. But now it is not, so please answer why.

**Comment by piere90** [comment, marked as answer]
You need to go here:

/resources/views/backend/includes/sidebar.blade.php

In this blade file PHP there are listed all the modules already present in-app, without your new module.
Copy one of the definitions like:

@php
            $module_name = "posts";
            $text = __('Posts');
            $icon = "fa-regular fa-file-lines";
            $permission = "view_".$module_name;
            $url = route('backend.'.$module_name.'.index');
        @endphp
        <x-backend.sidebar-nav-item :permission="$permission" :url="$url" :icon="$icon" :text="$text" />
        
And change the name of your new module;)

**Reply by nasirkhan** [reply]
thank you for replying.

**Reply by Chamila134** [reply]
According to user sidebar is not change and also there are some error display in some controller use, notification undefined like that. Can you help me

**Comment by nasirkhan** [comment]
Earlier a Menu Manager package was used for adding new modules automatically but unfortunately, that package has not been updated for a while and it has issues with localization too. So it had to be removed.
 
I am planning to use the database to store menu items, but I have not started working on it. If anyone is interested in working on this feature please submit a PR, it would be a great help.
